File: backend/src/main/java/top/vuhe/entity/equipment/dao/ChannelDao.java
package top.vuhe.entity.equipment.dao;

import lombok.*;
import top.vuhe.entity.BaseEntity;

/**
 * 状态
 * @author zhuhe
 */
@Data
@EqualsAndHashCode(callSuper = true)
public class ChannelDao extends BaseEntity {
    private String perUnit;
    private Integer currentState;
    private Integer currentValue;
    private String channelName;
}

File: backend-kt/src/main/kotlin/top/vuhe/sw/entity/equipment/dao/ChannelDAO.kt
package top.vuhe.sw.entity.equipment.dao

/**
 * 通道信息
 * 对应数据库中的表
 *
 * @property id           id
 * @property perUnit      单位
 * @property currentState 当前状态
 * @property channelName  通道名称
 */
data class ChannelDAO(
    val id: Int?,
    val perUnit: String?,
    val currentState: Int,
    val channelName: String?
)
